bindings_ffi/src/mls.rs

```diff
@@ -714,14 +714,14 @@ impl FfiConversations {
             _ => None,
         };
 
-        let convo = self
-            .inner_client
-            .create_group(group_permissions, metadata_options)?;
-        if !account_addresses.is_empty() {
-            convo
-                .add_members(&self.inner_client, account_addresses)
-                .await?;
-        }
+        let convo = if account_addresses.is_empty() {
+            self.inner_client
+                .create_group(group_permissions, metadata_options)?
```

Comment:
Should we just remove this method entirely and serve this use-case by passing an empty vec to `create_group_with_members`? Saves the code duplication here and below

Reply:
One difference is that `create_group_with_members` is async and the old one is sync.

I haven't dug in to how annoying of an update it would be.

I'll make a ticket for us to consolidate. Agree we don't need two methods in theory.